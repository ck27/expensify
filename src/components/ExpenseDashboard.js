import React from "react";
import { Link } from "react-router-dom";
import ExpenseList from "./ExpenseList";

const ExpenseDashboard = () => (
    <div>
        <h1>Expense Dashboard</h1>

        <Link to="/edit/1">Expense : 1</Link>
        <ExpenseList />
    </div>
);

export default ExpenseDashboard;