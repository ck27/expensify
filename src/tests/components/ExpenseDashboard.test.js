import React from "react";
import {shallow} from "enzyme";
import ExpenseDashboard from "../../components/ExpenseDashboard";

test("it should render ExpenseDashboard", () => {
    const content = shallow(<ExpenseDashboard />);
    expect(content).toMatchSnapshot();
})