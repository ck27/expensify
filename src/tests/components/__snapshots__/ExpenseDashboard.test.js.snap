// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render ExpenseDashboard 1`] = `
<div>
  <h1>
    Expense Dashboard
  </h1>
  <Connect(ExpenseFilter) />
  <Connect(ExpenseList) />
</div>
`;
